js/video/Example.svelte

```diff
@@ -38,7 +38,7 @@
 		/>
 	</div>
 {:else}
-	<div>{value}</div>
```

Comment:
In this case we see a black box - I think it would be better to omit it entirely to be consistent with image and the other components

<img width="1261" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/5b405a36-dace-4485-8136-22a8babd8286">

Reply:
Makes sense